libs-scala/db-utils/src/main/scala/com/digitalasset/http/dbbackend/DBConfig.scala

```diff
@@ -163,8 +166,10 @@ object JdbcConfig
     tablePrefix <- optionalStringField("tablePrefix").map(_ getOrElse "")
     maxPoolSize <- optionalIntField("poolSize").map(_ getOrElse PoolSize.Production)
     minIdle <- optionalIntField("minIdle").map(_ getOrElse MinIdle)
-    connTimeout <- optionalLongField("connectionTimeout").map(_ getOrElse ConnectionTimeout)
-    idleTimeout <- optionalLongField("idleTimeout").map(_ getOrElse IdleTimeout)
+    connTimeout <- optionalLongField("connectionTimeout")
+      .map(x => x.map(_.millis) getOrElse ConnectionTimeout)
+    idleTimeout <- optionalLongField("idleTimeout")
+      .map(x => x.map(_.millis) getOrElse IdleTimeout)
```

Comment:
```suggestion
    connTimeout <- optionalLongField("connectionTimeout").map(_ getOrElse ConnectionTimeout.millis)
    idleTimeout <- optionalLongField("idleTimeout").map(_ getOrElse IdleTimeout.millis)
```
I'm not 100% this compiles (or that it passes `scalafmt`), but if it does it looks more compact and readable.

Reply:
IIUC this will keep the values as a `long` whereas we want to convert the input values to a `FiniteDuration`.